### PR TITLE
docs(changelog): update for release 14.6.1 [skip ci]

### PR DESCRIPTION
**Pull request by amine-benselim**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+#### 🐛 Type: Bug
+
+- `application-shell`, `permissions`
+  - [#1009](https://github.com/commercetools/merchant-center-application-kit/pull/1009) CON-187: properly pass dataFences from menu to `Authorized.js` ([@amine-benselim](https://github.com/amine-benselim))
```
Comment (tdeekens) [inline CHANGELOG.md]: I would avoid having ticket numbers on public repos. I can deal with it in PRs but not really in release notes :)

**Comment by montezume** [comment]
I'm wondering why the package.json are changed in this PR? Did you release from this branch? I thought we usually merged the PR first and then released from master?

See [here](https://github.com/commercetools/merchant-center-application-kit/pull/1007) for example

**Comment by emmenko** [comment]
Yeah he released from this branch...I guess the instructions were not 100% clear. It's fine I guess, we just need to merge the PR

**Comment by tdeekens** [comment]
CleanShot 2019-08-29 at 10 55 54@2x

Yup the release exists. Let's just merge this when the ticket number is removed.

**Comment by tdeekens** [comment]
...or not.

**Comment by emmenko** [comment]
We can remove the ticket number as a follow up, the released packages already include the changelog so there is no point in changing it here anymore

**Comment by emmenko** [comment]
FYI: #1011

**Comment by tdeekens** [comment]
I know that they're published I just wanted to spare us another PR and change it here :0

**Pull request merged**